ISSNow/iss_now.go

```diff
@@ -0,0 +1,89 @@
+package ISSNow
+
+import (
+	"encoding/json"
+	"github.com/kelvins/geocoder"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"strconv"
+	"time"
+)
+
+type IssRaw struct {
+	Timestamp   int `json:"timestamp"`
+	IssPosition struct {
+		Latitude  string `json:"latitude"`
+		Longitude string `json:"longitude"`
+	} `json:"iss_position"`
+	Message string `json:"message"`
+}
+
+type Iss struct {
+	Latitude  string
+	Longitude string
+	Location  string
+}
+
+var (
+	url    = "http://api.open-notify.org/iss-now.json"
+	client = http.Client{
+		Timeout: time.Second * 10,
+	}
+)
```

Comment:
```suggestion
const (
    url = "http://api.open-notify.org/iss-now.json"
    clientTimeout = time.Second * 10
)
```
Do not use global variable unless you have to. The client does not need to be here.